pkg/drivers/hyperv/hyperv_windows.go

```diff
@@ -74,6 +74,7 @@ func (d *Driver) UpdateConfigRaw(rawConfig []byte) error {
 		}
 	}
 	if newDriver.DiskCapacity != d.DiskCapacity {
+		//#nosec G115 -- validation make sure disk value is in range
```

Comment:
I'd prefer if code in `pkg/drivers` did not make assumptions regarding how it's used, and in particular that we don't rely on code in `pkg/crc` having to be used before passing data to the drivers.
Moreover, I'm not sure we have the needed range check in validation